SimplenoteWidgets/Models/Note+CoreDataProperties.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import CoreData
+
+
+extension Note {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Note> {
+        return NSFetchRequest<Note>(entityName: "Note")
+    }
+
+    @NSManaged public var content: String?
+    @NSManaged public var creationDate: Date?
+    @NSManaged public override var isDeleted: Bool
+    @NSManaged public var lastPosition: NSNumber?
+    @NSManaged public var modificationDate: Date?
+    @NSManaged public var noteSynced: NSNumber?
+    @NSManaged public var owner: String?
+    @NSManaged public var pinned: NSNumber?
+    @NSManaged public var publishURL: String?
+    @NSManaged public var remoteId: String?
+    @NSManaged public var shareURL: String?
+    @NSManaged public var systemTags: String?
+    @NSManaged public var tags: String?
+
+}
```

Comment:
Since `Note+CoreDataClass.swift` + `Note+CoreDataProperties.swift` are meant to be a workaround, and aren't supposed to contain code in the class level, mind collapsing that to `Note+Widget.swift` maybe?

Background:
When using code gen, Core Data splits your NSManagedObject classes in two files. One is the concrete class definition, the other is an extension with the properties.

Goal is: if you need to throw in custom code into the Class, you just do so in `+CoreDataClass`. Whenever the model is updated, and you need to regen the files, your code is safe there (and shouldn't be overwritten).

Since we won't happen, let's collapse them both. **Plus:** please also add a note mentioning why this is required